src/dbg/value.cpp

```diff
@@ -1627,6 +1630,58 @@ bool valfromstring_noexpr(const char* string, duint* value, bool silent, bool ba
             if(new_size < read_size)
                 read_size = new_size;
         }
+        else if (string[0] == 'b'

```

Comment:
Shouldn't this be called after checking for `fs` and such? Because this could read from unknown memory right?